app/src/context/theme.ts

```diff
@@ -0,0 +1,115 @@
+import { useCallback } from "react";
+import { darkColors, lightColors } from "@fuel-ui/css";
+import { useFuelTheme } from "@fuel-ui/react";
+
+interface ColorMapping {
+  light: string;
+  dark: string;
+}
+export type ColorName =
+  | "chrome"
+  | "black1"
+  | "white1"
+  | "white2"
+  | "white3"
+  | "white4"
+  | "gray1"
+  | "gray2"
+  | "gray3"
+  | "gray4"
+  | "gray5"
+  | "gray6"
+  | "gray7"
+  | "gray8"
+  | "sgreen1"
+  | "disabled1"
+  | "disabled2"
+  | "disabled3";
+
+const COLORS: Record<ColorName, ColorMapping> = {
+  chrome: {
+    light: "chrome",
+    dark: "tomorrow_night_bright",
+  },
+  black1: {
+    light: darkColors.black,
+    dark: lightColors.gray4,
+  },
+  white1: {
+    light: lightColors.whiteA5,
+    dark: darkColors.scalesGreen2,
+  },
+  white2: {
+    light: lightColors.white,
+    dark: darkColors.scalesGray1,
+  },
+  white3: {
+    light: "#00000099",
+    dark: darkColors.gray12,
+  },
+  white4: {
+    light: lightColors.gray2,
+    dark: darkColors.black,
+  },
+  gray1: {
+    light: darkColors.gray6,
+    dark: lightColors.white,
+  },
+  gray2: {
+    light: darkColors.gray6,
+    dark: darkColors.gray11,
+  },
+  gray3: {
+    light: darkColors.gray6,
```

Comment:
nit: I noticed these are all mapping `darkColors.gray6` to a different color in dark mode. Should they all be the same? Perhaps, just keep `gray2`?